test/fixtures/p2pk.json

```diff
@@ -5,6 +5,7 @@
       "arguments": {
         "output": "030000000000000000000000000000000000000000000000000000000000000001 OP_CHECKSIG"
       },
+      "options": {},
       "expected": {
```

Comment:
did you merely add this to every vector?

Reply:
Yes.